OrbitGl/TimeGraph.cpp

```diff
@@ -726,41 +720,34 @@ void TimeGraph::SortTracks() {
   }
 }
 
-//----------------------------------------------------------------------------
-void TimeGraph::OnLeft() {
+const TextBox* TimeGraph::FindPrevious() {
```

Comment:
This is a very specialized method that works on the currently selected text box. Consider passing in the text box as a parameter or renaming to something like FindPreviousFromCurrentSelection() .